yeonjae/programmers-1/나누어떨어지는숫자배열.js

```diff
@@ -0,0 +1,12 @@
+function solution(arr, divisor) {
+  const answer = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] % divisor === 0) answer.push(arr[i]);
+  }
+  return answer.length === 0
+    ? [-1]
+    : answer.sort(function (a, b) {
+        return a - b;
+      });
+}
```

Comment:
```suggestion
function solution(arr, divisor) {
  const answer = [];
    
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % divisor === 0) answer.push(arr[i]);
  }
  
  answer.sort((a, b) => a - b);
  
  return !answer.length ? [-1] : answer
}
```
- 고생하셨습니다 👍 그런데 배열 고차 함수의 콜백함수에는 함수 선언문보다 화살표 함수를 사용해주세요 
- 그리고 sort는 사이드이펙트가 있는 메서드입니다. 즉, 기존 배열의 값을 바꿔버리죠 따라서 return 부분보다는 위에서 쓰는 것도 나쁘지 않을 것 같습니다. 물론 이건 코딩스타일 차이입니다